docs/api-reference/next/server.md

```diff
@@ -88,6 +88,23 @@ The introduction of the `307` status code means that the request method is prese
 
 The `redirect()` method uses a `307` by default, instead of a `302` temporary redirect, meaning your requests will _always_ be preserved as `POST` requests.
 
+### Accessing environment variables
```

Comment:
Given the above heading is a question - should we do "How do I access Environment Variables?"

Reply:
Moved to inline suggestion